src/components/tooltip.js

```diff
@@ -1,47 +1,36 @@
-import React, { useState, useEffect } from 'react'
-
-const Tooltip = ({ text, overrideClass = '', children, style }) => {
-  const [hover, setHover] = useState(false)
-  let clearDelay = null
-
-  useEffect(() => {
-    window.addEventListener('keydown', handleKeyDown)
-
-    return () => {
-      window.removeEventListener('keydown', handleKeyDown)
-    }
-  }, [])
-
-  const onMouseEnter = () => {
-    clearTimeout(clearDelay)
-
-    clearDelay = setTimeout(() => {
-      setHover(true)
-    }, 600)
-  }
-
-  const onMouseLeave = () => {
-    clearTimeout(clearDelay)
-    setHover(false)
-  }
-
-  const handleKeyDown = () => {
-    clearTimeout(clearDelay)
-    setHover(false)
-  }
-
+import React from 'react'
+import {
+  useTooltipState,
+  Tooltip as ReakitTooltip,
+  TooltipReference,
+} from 'reakit/Tooltip'
+
+const Tooltip = ({
+  text,
+  children,
+  overrideClass = '',
+  style,
+  isVisible = false,
```

Comment:
this is what breaks the tooltips, remove it

Reply:
if i remove it the tooltips won’t display on `Slider` drag  i'lo try an other solution today !